content/authorization/guides/system-vendor/consent/liststatuschanges/_index.en.md

```
@@ -0,0 +1,75 @@
+```
+
+| Response Field | Description |
+|----------------|-------------|
+|links.Next|URL to fetch the next page. null if there are no more results.|
+|data[].consentRequestId|Unique identifier of the consent request.|
+|data[].eventType|the type of status event f.eks accepted, rejected, revoked, deleted, used. Event created will not be listed as it is not a status change.|
+|data[].changedDate| the timestamp of when the status change occured|
+
+
+**Error Responses**
+| StatusCode | Description |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)